src/arch/arm/64/kernel/vspace.c

```diff
@@ -1071,45 +1081,36 @@ pude_t *pageDirectoryMapped(asid_t asid, vptr_t vaddr, pde_t *pd)
 
 #ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
 
-static void invalidateASID(asid_t asid)
+static asid_map_t *findMapRefForASID(asid_t asid)
```

Comment:
There is is this convention that the functions that expect things to maybe not be there are called `find` and return a potential failure. This one is more a `get`, because it asserts the presence of the ASID pool. I currently have the same `find` name in the prelim spec, based on Curtis' original commit and probably going back to x64, and it makes sense to mimic `findMapForASID`, but overall I think `getMapRefForASID` would be a better name.

Reply:
Actually, I lie, I don't have this function at all in the spec -- I need to unfold this in Haskell because there is no direct ref to a position in an array there. Anyway, still prefer the `get` name :-)

Reply:
Ok, get makes more sense.